guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,436 @@
+
+Hardware to support AI for a combined workload for the modalities has traditionally varied. In general, IT infrastructure will assess the mix of modalities and clinical workloads based on the use cases to be supported.  Next, the appropriate mix and utilization method based on these clinical workloads will determine what hardware and compute resources are deployed. 
+
+Typical hardware specifications to be used for the computational workloads are provided below based on specifications seen in the marketplace for AI deployments.
+
+
+### Hardware Targets {#hardware-targets}
+| System   | Hardware Specifications for AI Platform (representative)                                                                                                                                                                                                                                                                                                                                                      |
+| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| System A | 1U, dual socket rack system supports up to: Two Intel Xeon Scalable processors. Twenty four DIMM slots. Storage capacity of four Graphical Processing Units. A100 80GB Two power supply units (PSUs) Optional two 2.5-inch cabled SATA SSDs installed only in PSU 2 bay.                                                                                                                                      |
+| System B | GPU Up to 4x NVIDIA GPUs RTX 30XX, RTX A6000, RTX A5000, RTX A4000, and Quadro RTX options Processor AMD Threadripper or Intel Core i9 Configurable up to 64 cores, 128 threads, and 256 MB cache Memory Up to 1 TB Fits up to eight 128 GB LRDIMMs at 3200 MHz OS drive Up to 2 TB 3,200 MB/s seq. read and 2,000 MB/s seq. write Extra storage Up to 61 TB Fits up to eight 7.68 TB SATA SSDs. Power supply |
+## Appendix {#appendix}
+
+
+### Clinical Workloads Out of Scope  {#clinical-workloads-out-of-scope}
+
+For the purpose of this document, the following workloads are out of scope:
+
+**Digital Pathology** 
+
+Analyze pathology images with AI algorithms.
+
+
+### Image Enhancement {#image-enhancement}
+
+Using computation and AI techniques, clarify the image for human and machine analysis.
+
+**Medical Text Analysis:**
+
+Natural language recognition used with medical notes producing coding and diagnostic assistance. For instance, validate that the medical regime is both safe and effective for this patient given known medical history. 
```

Comment:
"Natural language processing" might be a more general expression other than "Natural language recognition".

Reply:
Agreed.  NLP is known. NLR??
Fixed